.env/mainnet.env

```
@@ -1,7 +1,7 @@
 CARGO_FEATURES_BUILD = "mainnet"
 CARGO_FEATURES_BUILD_MIGRATION = "mainnet,migration"
 CARGO_FEATURES_BUILD_TEST = "mainnet,integration-test,migration"
-CARGO_FEATURES_TEST = ""
+CARGO_FEATURES_TEST = "migration_tests"
```

I'd use a dash here rather than an underscore.

```suggestion
CARGO_FEATURES_TEST = "migration-tests"
```